modules/ossm-federation-config-meshPeer.adoc

```diff
@@ -0,0 +1,109 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-task_{context}"]
+= Joining a federated mesh
+
+The federation between two meshes is declared by a `ServiceMeshPeer` resource that defines the federation from one mesh to the other and is used to configure discovery for the peer mesh, access to the peer mesh, and certificates used to validate the other mesh’s clients.
+
+//Insert ServiceMeshPeer diagram here
+
+Configuration in the `ServiceMeshPeer` resource includes the following:
+
+* The address of the other mesh’s ingress gateway, which will be used for discovery and service requests.
+* The names of the local ingress and egress gateways that will be used for interactions with the specified peer mesh.
+* The client ID used by the other mesh when sending requests to this mesh.
+* The trust domain used by the other mesh.
+* The name of a `ConfigMap` containing a root certificate that will be used to validate client certificates in the trust domain used by the other mesh
+
+Meshes are federated on a one-to-one basis, so each pair of peers requires a `ServiceMeshPeer` file specifying the federation connection to the other service mesh.  For example, federating three meshes named `red`, `blue`, and `green` would require six `ServiceMeshPeer` files.
+
+. On red-mesh-system, create a `ServiceMeshPeer` for the green mesh
+. On red-mesh-system, create a `ServiceMeshPeer` for the blue mesh
+. On green-mesh-system, create a `ServiceMeshPeer` for the red mesh
+. On green-mesh-system, create a `ServiceMeshPeer` for the blue mesh
+. On blue-mesh-system, create a `ServiceMeshPeer` for the red mesh
+. On blue-mesh-system, create a `ServiceMeshPeer` for the green mesh
+
+In the following example, the administrator for the `red-mesh` is configuring federation with the `green-mesh`.
+
+.Example ServiceMeshPeer resource for red-mesh
+[source,yaml]
+----
+kind: ServiceMeshPeer
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: green-mesh
+  namespace: red-mesh-system
+spec:
+  remote:
+    addresses:
+    - ingress-red-mesh.red-mesh-system.apps.domain.com
+  gateways:
+    ingress:
+      name: ingress-green-mesh
+    egress:
+      name: egress-green-mesh
+  security:
+    trustDomain: green-mesh.local
```

Comment:
@JStickler this line is not correct. It should be this line trust and then next line domain: green-mesh.local.
You can compare that spec with your previous ones above. The correct path should be spec.security.trust.domain